pkg/formats/bencode_test.go

```diff
@@ -30,17 +34,19 @@ func TestBencodeUnmarshal(t *testing.T) {
 		output string
 		err    error
 	}{
-		{`{"hi":"hi"}`, "d2:hi2:hie", nil},
+		{`{"hi":"hi"}`, "d2:hi2:hie\n", nil},
```

Comment:
Note that currently each encoder is writing a newline now, as indicated by this changed test. This is because the encoder handles writing output, and newlines are part of that. Previously, `printValue` did this.